java/src/Ice/src/main/java/com/zeroc/Ice/ConnectionI.java

```diff
@@ -2282,17 +2282,13 @@ private boolean sendResponseImpl(OutputStream outputStream, boolean isTwoWay, by
             notifyAll();
           }
 
-          if (_state >= StateClosed) {
-            assert (_exception != null);
-            throw (LocalException) _exception.fillInStackTrace();
```

Comment:
I find this old code very incorrect.
We were overwriting the stack trace in _exception with the current stack trace, which makes no sense.